app/models/items/voucher.rb

```diff
@@ -104,7 +104,7 @@ def item_description
 
   scope :open, -> { where(:checked_in => false).where(:showdate => nil) }
 
-  def unreserved? ; showdate_id.to_i.zero?  ;  end
+  def unreserved? ; showdate_id.to_i.zero? | showdate_id.blank? ;  end
```

Comment:
this is wrong.  `|` is almost certainly the wrong operator.  and `showdate_id.to_i.zero?` will do the right thing if `showdate_id` is blank so the second clause isn't necessary